### PR TITLE
ref(source-stripe): remove stripe py-package

### PR DESCRIPTION
**Pull request by artem1205**
## What

refactor to remove python package `stripe`

Reason: 
- bypass SSL limitation on internal regression tests.
- used only in `check` and can be replaced with existing `HttpStream` implementation.

## How

remove python stripe package

## Review guide

1. `airbyte-integrations/connectors/source-stripe/source_stripe/source.py`

## Can this PR be safely reverted and rolled back?

- [ ] YES 💚
- [ ] NO ❌

### CODE REVIEW

**Comment by artem1205** [comment]
Regression tests failed only for control version.

File: airbyte-integrations/connectors/source-stripe/source_stripe/source.py [hunk 1]
```diff
@@ -107,12 +108,26 @@ def validate_and_fill_with_defaults(config: MutableMapping[str, Any]) -> Mutable
...
+        account_stream = StripeStream(name="accounts", path="accounts", use_cache=USE_CACHE, **args)
+        availability_strategy = HttpAvailabilityStrategy()
         try:
-            stripe.Account.retrieve(config["account_id"])
-        except (stripe.error.AuthenticationError, stripe.error.PermissionError) as e:
-            return False, str(e)
+            stream_is_available, reason = availability_strategy.check_availability(account_stream, logger)
+            if not stream_is_available:
+                return False, reason
+        except Exception as error:
```
Comment (maxi297) [inline airbyte-integrations/connectors/source-stripe/source_stripe/source.py:128]: If we catch all the exceptions here, we might hide some system errors that should have trigger paging. We should probably do something similar as [this PR](https://github.com/airbytehq/airbyte/pull/45205) where we only catch AirbyteTracedException of type config error

File: airbyte-integrations/connectors/source-stripe/source_stripe/source.py [hunk 2]
```diff
@@ -107,12 +108,26 @@ def validate_and_fill_with_defaults(config: MutableMapping[str, Any]) -> Mutable
         return config
 
     def check_connection(self, logger: logging.Logger, config: MutableMapping[str, Any]) -> Tuple[bool, Any]:
-        self.validate_and_fill_with_defaults(config)
-        stripe.api_key = config["client_secret"]
+        config = self.validate_and_fill_with_defaults(config)
```
Comment (maxi297) [inline airbyte-integrations/connectors/source-stripe/source_stripe/source.py:111]: nit: the logic to prepare the config seems very similar than the one in `streams`. Should we extract that?

**Review by maxi297 — COMMENTED** [review]
I think we should ensure that we don't prevent alerting when needed. The rest looks good to me

**Review by maxi297 — APPROVED** [review]
LGTM!

**Comment by tcboles** [comment]
This PR does not allow for tokens that do not have read access to all platform connected accounts. I am getting the following error.

The endpoint https://api.stripe.com/v1/accounts?limit=100 returned 400: Bad Request. You cannot access the connected accounts of your platform's connected accounts..  Please visit https://docs.airbyte.com/integrations/sources/stripe to learn more.  You cannot access the connected accounts of your platform's connected accounts.

**Pull request merged**